File: CarOrderApiSln/CarOrderApi/Model/Order.cs
namespace CarOrderApi.Model
{
    public class Order

    {
        public int Id { get; set; }
        public string UserId { get; set; }
        public int CarId { get; set; }

        public decimal TotalPrice { get; set; }
        public DateTime OrderDate { get; set; }
        public string Status { get; set; } = "Panding";

        public string ShippingAddress { get; set; }
        public string PhoneNumber { get; set; }
        public ApplicationUser User { get; set; }
        public Car Car { get; set; }
        public int Quantity { get; set; }
    }
}

File: CarOrderClient/src/app/models/Order.ts
import { Car } from "./car";

export interface Order {
  orderId: number;
  carId: number;
  carName: string;
  brand: string;
  orderDate: string;
  status: string;
  carImage: string;
  totalprice:number;
  quantity:number;
  shippingAddress:string;
  phoneNumber:string;
  paymentMethods:string;
  car:Car;

}